src/lib/ledger-context.js

```diff
@@ -35,12 +35,17 @@ function parseAndValidateLedgerUrls (metadataUrls) {
 }
 
 class LedgerContext {
-  constructor (host, ledgerMetadata) {
+  constructor (host, ledgerMetadata, overrides) {
+    const metadata = Object.assign({}, ledgerMetadata, overrides || {})
```

Comment:
Yeah, that was my thought. I think you should just be able to overwrite whatever you want